tests/functional/test_freeze.py

```diff
@@ -555,7 +555,7 @@ def test_freeze_with_requirement_option_package_repeated_multi_file(script):
     """)
     expected_out += _freeze_req_opts
     expected_out += "## The following requirements were added by pip freeze:"
-    expected_out += os.linesep + textwrap.dedent("""\
+    expected_out += '\n' + textwrap.dedent("""\
```

Comment:
Actually, while we're at it, maybe changing these to:

```py
expected_out = textwrap.dedent("""
    simple == 1.0
    {}
    ## The following requirements were added by pip freeze:
    ...meta==1.0...
""").format(_freeze_req_opts)
```

is a good change?

Reply:
We could, yes, do you want me to make the change and amend the other tests too?

Reply:
Actually, let's fix the build for now. I'll make a follow up PR to clean up the test. :)

Reply:
Here's the [fixup](https://github.com/benoit-pierre/pip/commit/57dfac71a9178192a1ab83025592d04ed0fe728c) if you change your mind.